core/services/workflows/engine_test.go

```diff
@@ -64,6 +64,35 @@ targets:
       abi: "receive(report bytes)"
 `
 
+// newTestEngine creates a new engine with some test defaults.
+func newTestEngine(t *testing.T, reg *coreCap.Registry, spec string) *Engine {
+	cfg := Config{
+		Lggr:     logger.TestLogger(t),
+		Registry: reg,
+		Spec:     spec,
+	}
+	eng, err := NewEngine(cfg)
+	require.NoError(t, err)
+	eng.xxxMaxRetries = 1
```

Comment:
As I said above, can you add this as private variables to Config, and use their absence to apply defaults we want to use in prod?

Reply:
Ah, I wanted to do this initially, it felt intuitive to me! I wanted to keep it consistent with the `xxx` pattern though. Happy to make this change